charts/datadog/templates/_orchestrator_explorer_config.yaml

```diff
@@ -38,14 +38,10 @@ If customResources is empty, then we don't add crd collection config.
 
 {{- if eq (include "orchestratorExplorer-add-crd-collection-config" .) "true" -}}
 orchestrator.yaml: |-
-  cluster_check: {{ .Values.clusterChecksRunner.enabled }}
   init_config:
   instances: 
-    - skip_leader_election: {{ .Values.clusterChecksRunner.enabled }}
-      {{- if gt (len $.Values.datadog.orchestratorExplorer.customResources) 0 }}
```

Comment:
we already checked the length here https://github.com/DataDog/helm-charts/blob/37f04e329a0894a4a5553138d5984ac99c95d1b9/charts/datadog/templates/_orchestrator_explorer_config.yaml#L27